### ISSUE TITLE
readthedocs not building

### ISSUE THREAD

**Issue by raphaeldussin** [xgcm/xmitgcm#131]
latest build is 4 months old.

**Comment by raphaeldussin** [comment]
@rabernat looks like the webhook is broken or something. Any ideas?

**Comment by rabernat** [comment]
I just messed around with the RTD settings and maybe fixed it.

**Comment by raphaeldussin** [comment]
yes, I see some new failed builds now. I can take it from there.

**Comment by rabernat** [comment]
I triggered those manually. To see if the new webhook works, we need a new commit on master.

**Comment by raphaeldussin** [comment]
got it! I am merging a new one with some fixes to the readthedocs.yml

**Comment by raphaeldussin** [comment]
merged commit and RTD doesn't build :(

**Comment by rabernat** [comment]
Was this resolved?

**Comment by raphaeldussin** [comment]
no, saving changes in advanced settings usually triggers a build but it doesn't respond to following code changes

**Comment by rabernat** [comment]
Perhaps if you read this carefully you can find a solution:
https://docs.readthedocs.io/en/stable/webhooks.html

**Comment by rabernat** [comment]
I believe I have fixed this now. Next time we push to master, the RTD docs should build.

**Comment by raphaeldussin** [comment]
what did you do to fix it? I replaced the webhook but I didn't have a PR to merge to test if it was working.
